3rdPartyCypress/cypress/integration/mediaAssetPagePrerollAds.js

```diff
@@ -0,0 +1,87 @@
+import getAppEnv from '../../../cypress/support/helpers/getAppEnv';
+import appToggles from '../../../cypress/support/helpers/useAppToggles';
+import { getEmbedUrl } from '../../../cypress/integration/pages/mediaAssetPage/helpers';
+
+const environment = getAppEnv();
+
+describe('Media Asset Pages - Preroll Ads', () => {
+  describe('should not load the preroll ad plugin', () => {
+    const mapsWithoutPrerollConfig = {
+      local: [],
+      test: [],
+      live: [
+        {
+          reason: 'service does not have preroll ads enabled',
+          paths: [
+            'https://www.bbc.com/pidgin/media-44221514', // CPS Video
+            'https://www.bbc.com/russian/media-52728860', // CPS Video
```

Comment:
We will need to remove this before Russian Go Live - not sure if it's worth keeping for now

Reply:
Agreed with Lily that we can just remove this, and will add to the `withPreroll` list once Russian MAPs go live